src/main/java/Database/Interaction/Presentation/Servlets/GENERIC_INFO.java

```diff
@@ -28,7 +28,7 @@
     public static final String SITE_URL =
             ConfigClass.RUNNING_ON_SERVER ?
                     "/dcs" :
-                    "/SecureDataCollector-1.0-SNAPSHOT";
```

Comment:
aky je teda ponovom definitivne zaklad url pre servlety web apky?

Reply:
Ja som to zaktualizoval v Postmanovi. Príklad
http://localhost:8080/api/measurements-add
http://localhost:8080/api/flat-add
http://localhost:8080/admin/view
http://localhost:8080/index
cize na locale http://localhost:8080/ a na serveri je zaklad
https://team14-20.studenti.fiit.stuba.sk/dcs/